daisy_workflows/image_build/windows/windows-build.wf.json

```diff
@@ -2,6 +2,9 @@
   "Vars": {
     "media": {"Required": true, "Description": "GCS or local path to Windows Server ISO."},
     "edition": {"Required": true, "Description": "Windows Server Edition to install."},
+    "family": {"Required": true, "Description": "Desired image family of the output image"},
```

Comment:
Don't pull this in via the workflow, name is essentially meaningless and family is an implementation detail.

Reply:
Sure.